src/common/reducers/socket.js

```diff
@@ -17,21 +17,25 @@ export default function socket(oldState = cloneDeep(defaultState), action) {
       return Object.assign(state, {connected: false, connecting: false});
 
     case socketActions.CHAT: {
+      const { chatMessages, userDataByClientId: userDataMap } = state;
+      const { sender, msg } = action.payload;
+
       const message = {
-        user: action.payload.sender ?
-          (state.userDataByClientId[action.payload.sender] ?
-            state.userDataByClientId[action.payload.sender].displayName :
-            action.payload.sender) :
-          'You',
-        text: action.payload.msg,
+        user: sender ?
+                (userDataMap[sender] ? userDataMap[sender].displayName : sender) :
+                'You',
```

Comment:
Should `'You'` be a constant?

Reply:
I guess I could make a `YOU_PLAYER_NAME` constant or something but I think it's pretty obvious what its value would be?

Reply:
What if it changes :O

Reply:
uhhhh